assets/sass/widgets/_googlesitekit-widget-analyticsAllTrafficV2.scss

```diff
@@ -99,8 +100,8 @@
 		}
 
 		h2 {
```

Comment:
This was already a problem before this PR, but I just noticed it: We should not use a `h2` in the component here as it results in incorrectly nested heading levels. Let's use a `div` like we do in the regular `DataBlock`s too.